doc/content/reference/adr/_index.md

```diff
@@ -36,33 +46,21 @@ The implementation is based on Semtech's recommended algorithm described in [thi
 5. If there's still margin after reaching the maximum data rate, [decrease the transmit power](https://github.com/TheThingsNetwork/lorawan-stack/blob/5a816e8171f993db9659566286d45725698f032e/pkg/networkserver/mac/adr.go#L273-L281)
 6. Depending on packet loss, [increase the number of retransmissions](https://github.com/TheThingsNetwork/lorawan-stack/blob/5a816e8171f993db9659566286d45725698f032e/pkg/networkserver/mac/adr.go#L288-L296)
 
-## Custom ADR Algorithm
-
-Besides {{% tts %}} ADR mechanism described [above]({{< ref "/reference/adr#the-things-stack-adr-algorithm" >}}), {{% tts %}} also supports using a custom ADR, meaning ADR parameters (data rate, Tx power, number of transmissions per uplink frame) can be controlled manually.
-
-{{< note >}} We recommend to test the process described below on test devices before implementing it in production. {{</ note >}}
+### Enable and Configure Dynamic Mode
 
-Before setting ADR parameters to desired values, you first need to turn off the default, {{% tts %}} ADR mechanism. To turn of {{% tts %}} ADR using the [CLI]({{< ref "/the-things-stack/interact/cli" >}}):
+To turn on {{% tts %}} ADR using the [CLI]({{< ref "/the-things-stack/interact/cli" >}}):
 
 ```bash
-ttn-lw-cli end-devices set --application-id <app-id> --device-id <dev-id> --mac-settings.use-adr=false
+ttn-lw-cli end-devices set --application-id <app-id> --device-id <dev-id> --mac-settings.adr.mode.dynamic
 ```
 
-After {{% tts %}} ADR mechanism is disabled, the Network Server will no longer try to optimize ADR parameters.
-
-Now you can manually set ADR parameters to desired values using the CLI:
+Values of parameters like data rate index, transmission power index and number of transmissions are calculated by the ADR algorithm, but their minimum and maximum values can be set by the user as follows:
 
 ```bash
-ttn-lw-cli end-devices set --application-id <app-id> --device-id <dev-id> --mac-state.desired-parameters.adr-data-rate-index <data_rate> --mac-state.desired-parameters.adr-tx-power-index <power_index> --mac-state.desired-parameters.adr-nb-trans <nb_trans>
+ttn-lw-cli end-devices set --application-id <app-id> --device-id <dev-id> --mac-settings.adr.mode.dynamic --mac-settings.adr.mode.dynamic.max-data-rate-index <data_rate> --mac-settings.adr.mode.dynamic.max tx-power-index <power_index> --mac-settings.adr.mode.dynamic.max-nb-trans <nb_trans> --mac-settings.adr.mode.dynamic.min-data-rate-index <data_rate> --mac-settings.adr.mode.dynamic.min tx-power-index <power_index> --mac-settings.adr.mode.dynamic.min-nb-trans <nb_trans>
```

Comment:
Do we have a mention somewhere that `<data rate>` is an index ? (so 0 is SF12 in EU868 for example)